src/Raven.Server/Dashboard/MachineResourcesNotificationSender.cs

```diff
@@ -21,13 +19,23 @@ public class MachineResourcesNotificationSender : BackgroundWorkBase
         private readonly TimeSpan _notificationsThrottle;
 
         private DateTime _lastSentNotification = DateTime.MinValue;
+        private static byte[][] _buffers;
```

Comment:
This is not valid. Multiple threads may call this, so you cannot use a single static buffer.

Reply:
There is a single instance of MachineResourcesNotificationSender

Reply:
And it is thread safe?
Doesn't matter what the case is right now, though, because we don't want to leave a trap here for the future.

Reply:
done